### ISSUE TITLE
Sequential: Font Awesome icon displayed inverted in Header

### ISSUE THREAD

**Issue by krutidugade** [Automattic/themes#55]
## Steps to replicate

- Activate Sequential theme
- Install Font Awesome fonts on AT/Business Site - https://fontawesome.com/get-started/web-fonts-with-css
- Go to https://wordpress.com/ > My Site(s) > Customize > Theme Options > Homepage
- Add the code in Top Area Content to display phone icon
<i class="fas fa-phone-square" data-fa-transform="flip-h"></i>
- Click Publish

## Result

The phone icon appeared inverted

## Expected

The phone icon should be appearing as it is.

The data-fa-transform="flip-h" attribute wasn't recognized on saving the code. I believe it was being stripped out.

* User report: #2228570-hc
* Follow-up ticket: #1048486-zd
* User's site: https://accidentlaw.com.au
* Slack - https://a8c.slack.com/archives/C03TY6J1A/p1522231430000232

**Comment by laurelfulford** [comment]
Noting that this appears to be temporarily fixed with Custom CSS.

**Comment by laurelfulford** [comment]
To recreate:

1. I installed a plugin that allowed me to add stuff to the `<head>` of my test site
2. I added FontAwesome as per the stylesheet `<link>` method mentioned here: https://fontawesome.com/get-started/web-fonts-with-css
3. I included `<i class="fas fa-phone-square" data-fa-transform="flip-h"></i>` in Customizer > Theme Options > Top Area Content 

I could see the same issue as reported. I could also recreate this issue with another theme that does something similar: in the Header Top Text area in Lodestar.

It looks like the sanitization function used for these fields -- `wp_kses_post` -- strips out the `data-fa-transform` attribute. This function is native to WordPress, so it's not a theme-specific. It's possible to change how sanitization is handled for this field in Sequential, but given that this hasn't come up before, and that it could be fixed via CSS for this user, I think it's best to leave as is.
